src/main/java/org/fenixedu/academic/domain/contacts/PartyContact.java

```diff
@@ -176,7 +177,9 @@ public void edit(final PartyContactType type, final boolean defaultContact) {
         setDefaultContactInformation(defaultContact);
     }
 
-    public abstract String getPresentationValue();
+    public String getPresentationValue() {
+        return PhoneUtil.resolve(this);
```

Comment:
Why make PartyContact depend on PhoneUtil?
It's a very strange dependency. The PartyContact class should not have to know anything about PhoneUtil.